int32 simDim
int32 simType
float64 simSlope
float64 maxForce
---
